Training_25/Program.cs

```diff
@@ -3,11 +3,33 @@
 // Copyright (c) Metamation India.
 // ------------------------------------------------------------------
 // Program.cs
-// Program on main branch.
+// Program returns the smallest number of steps taken to transform the number into identical digits
 // ------------------------------------------------------------------------------------------------
+using static System.Console;
 namespace Training_25;
 internal class Program {
-   static void Main (string[] args) {
-      Console.WriteLine ("Hello, World!");
+   static void Main () {
+      int num;
+      while (true) {
+         Write ("Enter a number: ");
+         if (int.TryParse (ReadLine () ?? "0", out num)) break;
+         WriteLine ("Invalid input. Please enter a valid number.");
+      }
+      WriteLine ($"{num} -> {MinSteps (num)} Steps");
+   }
+
+   //Returns the minimum number of steps needed to convert a number into identical digits

```

Comment:
Leave a space after //.
```suggestion
   // Returns the minimum number of steps needed to convert a number into identical digits

```